lms/envs/common.py

```diff
@@ -1733,6 +1733,9 @@
     'cors_csrf',
 
     'commerce',
+
+    # Bookmarks API
+    'bookmarks_api',
```

Comment:
i thought it's kind of convention that we are using for such apps, just like we are doing for https://github.com/edx/edx-platform/tree/master/lms/djangoapps/course_structure_api